src/sass/components/_gallery.scss

```diff
@@ -0,0 +1,258 @@
+.section--gallery {
+  padding: 50px 0;
+
+  .container {
+    .row {
+      .gallery {
+        .row {
+          .gallery-box {
+            .tabs {
+              .row {
+                .tab:nth-child(4) {
+                  border-width: 1px;
+                }
+
+                .tab {
+                  background-color: #fff;
+                  text-transform: uppercase;
+                  border: solid;
+                  border-width: 1px 0 1px 1px;
+                  border-color: $gallery-border;
+                  padding: 10px;
+                  text-align: center;
+                  font-weight: 600;
+
+                  &:hover {
+                    cursor: pointer;
+                    border-width: 2px 1px 0 1px;
+                    border-top-color: $primary;
+                    color: $primary;
+                  }
+                }
+
+                .active {
+                  background-color: $panel-bar;
+                  border-width: 2px 1px 0 1px;
+                  border-top-color: $primary;
+                  color: $primary;
+                }
+              }
+            }
+
+            .img-box {
+              background-color: $panel-bar;
+              position: relative;
+
+              .row {
+                position: relative;
+
+                .img-box-background {
+                  width: 100%;
+                  object-fit: cover;
+                  max-height: 400px;
+                }
+
+                .outlines {
+                  position: absolute;
+                  top: 30%;
```

Comment:
Jak to ma być na środku w pionie to nie rób tego na oko (30%) tylko odsuń o 50% i potem zastosuj translate -50% w pionie żeby o połowę swojej wysokości przesunąć tooltipy do góry (wtedy 50% top to będzie idealny środek)